rest-server/src/services/data_access_object.cpp

```diff
@@ -59,4 +59,18 @@ bool DataAccessObject::RemoveUser(User user) {
 void DataAccessObject::AddUser(User user) {
     users__.push_back(user);
 }
+
+void DataAccessObject::UpdateUser(int id, web::json::object user_info) {
```

Comment:
When I try to update users I have added via the POST command, I am not able to update them with a PUT command... Not sure what's up with that, it very well could be an issue with my POST code or something else. It looks fine and I'm able to update the seeded data (Shawn and Kris). Going to test later today and find the root cause.

Reply:
OK, this makes NO sense to me. I put in logging statement, rebuilt it, ran it, and it's fine now.... 

So, I approve now. Something must have been wrong on my end, maybe I had a staged file from another branch messing it up.